public/.htaccess

```diff
@@ -0,0 +1,13 @@
+# rewrite ^/(.*)/$ /$1 permanent;
```

Comment:
Could you please use `Nginx` instead of `Apache`?

Reply:
I think that's alright to have `.htaccess`. Nginx doesn't require or use it but Apache users won't have to add it. Deleting it is easy.